src/server/req_holdjob.c

```diff
@@ -405,26 +423,27 @@ post_hold(struct work_task *pwt)
 		conn->cn_authen &= ~PBS_NET_CONN_NOTIMEOUT;
 	}
 
-	pjob = find_job(preq->rq_ind.rq_hold.rq_orig.rq_objname);
 	if (pjob  == NULL) {
 		log_event(PBSEVENT_DEBUG, PBS_EVENTCLASS_JOB, LOG_DEBUG,
 			preq->rq_ind.rq_hold.rq_orig.rq_objname,
 			msg_postmomnojob);
-		if (preq->rq_nest)
-			reply_preempt_jobs_request(PBSE_SYSTEM, 0, preq);
+		if (pjob->ji_pmt_preq != NULL)
+			reply_preempt_jobs_request(PBSE_SYSTEM, PREEMPT_METHOD_CHECKPOINT, pjob);
```

Comment:
This code was incorrect.  It was in a block of if (pjob == NULL).  This would have dereferenced pjob and crashed.  I removed it.  We shouldn't get here since we catch the case where the job is not there in req_preemptjob()